### PR TITLE
Backend/homepage/side dish

### PR DESCRIPTION
**Pull request by hiroiY**
I  created the side dish page on the homepage.

added :
- the function getSidedishPosts() in "HomeController"
- New route in "web.php"
   /home/sidedish'
   /home/recently' -> Because  I cannot see the recently shared page I added that route.

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -11,7 +11,9 @@
 Auth::routes();
 // require __DIR__ . '/auth.php';
 Route::get('/home', [App\Http\Controllers\HomeController::class, 'index'])->name('home');
+Route::get('/home/recently', [App\Http\Controllers\HomeController::class, 'index']);
 Route::get('/home/appetizer', [App\Http\Controllers\HomeController::class, 'index']);
+Route::get('/home/sidedish', [App\Http\Controllers\HomeController::class, 'index']);
```
[the remark this answers is not in the document]
Comment (kredo-school) [reply routes/web.php:16]: check also home/appetizer

[the remark this answers is not in the document]
Comment (hiroiY) [reply routes/web.php:16]: Yes. I made that in same function index. 
Also, I'm going to create the main_dish and dessert page in the same function index.
These use same tabs and blade file that why I use same function. 
Should I separate the function on each page?

Screenshot 2024-06-25 214138

[the remark this answers is not in the document]
Comment (kredo-school) [reply routes/web.php:16]: Can you try to comment these 3 lines  and check if it works the same without these?

```
Route::get('/home/recently', [App\Http\Controllers\HomeController::class, 'index']);
Route::get('/home/appetizer', [App\Http\Controllers\HomeController::class, 'index']);
Route::get('/home/sidedish', [App\Http\Controllers\HomeController::class, 'index']);
```

[the remark this answers is not in the document]
Comment (hiroiY) [reply routes/web.php:16]: Do you mean trying to comment out?

[the remark this answers is not in the document]
Comment (hiroiY) [reply routes/web.php:16]: I tried to comment out these It doesn't work without these.
image

[the remark this answers is not in the document]
Comment (hiroiY) [reply routes/web.php:16]: Each tab page is inserted into the homepage blade file (views/homepage/homepage.blade.php).

So the reason there, is it doesn't work without those?

**Pull request merged**